GVFS/GVFS.Service/GvfsService.cs

```diff
@@ -55,12 +54,21 @@ public void Run()
 
                     using (ITracer activity = this.tracer.StartActivity("EnsurePrjFltHealthy", EventLevel.Informational))
                     {
+                        // Make a best-effort to enable PrjFlt. Continue even if it fails.
+                        // This will be tried again when user attempts to mount an enlistment.
                         string error;
                         EnableAndAttachProjFSHandler.TryEnablePrjFlt(activity, out error);
                     }
 
                     this.serviceStopped.WaitOne();
                 }
+
+                // https://github.com/Microsoft/VFSForGit/issues/774.

```

Comment:
I don't think there's any need to include the issue number that prompted this change.  You've done a good job of explaining the scenario in the rest of the comment.